src/annotator/notebook.js

```diff
@@ -105,6 +108,22 @@ export default class Notebook extends Delegator {
     this.container.className = 'notebook-outer';
     shadowRoot.appendChild(this.container);
 
+    render(
+      <div className="Notebook__controller-bar">
+        <Button
```

Comment:
This `Button` might benefit from using an extra prop; I think it might be `title`, that can allow a more verbose `aria-label` to be set. In this case, I might suggest setting `title` (double-check that prop name, please) to `Close the Notebook`.

Reply:
You are right, it is `title`. I added that.